docs/notebooks/data_transformation.pct.py

```diff
@@ -244,6 +244,89 @@ def normalise(x, mean=None, std=None):
 print(f"observation: {observations[arg_min_idx, :]}")
 
 
+# %% [markdown]
+#
+# Now lets try managing data transformation with a `NormalizationMixin`.
+#
+# First, we subclass the mixin to handle our normalization. See `trieste.models.normalization.StandardizationMixin` for an example that normalizes both query points and observations to have zero mean and unit variance. We modify this further to add support for updating the priors over hyperparameters.
+#
+# We then create our new model class that handles normalization automatically.
+
+# %%
+from trieste.models.normalization import StandardizationMixin
+
+class StandardizedGaussianProcessRegression(StandardizationMixin, GaussianProcessRegression):
+    def __init__(self, dataset, model, optimizer, num_kernel_samples):
+        super().__init__(dataset, model, optimizer, num_kernel_samples)
+
```

Comment:
This seems quite complicated for a standard user.  Maybe we could just have a normalizer input to all our model classes?